application/models/organization_test.go

```diff
@@ -499,3 +407,46 @@ func (ms *ModelSuite) TestOrganization_GetUsers() {
 		})
 	}
 }
+
+func (ms *ModelSuite) TestOrganization_AllWhereUserIsOrgAdmin() {
+	t := ms.T()
+
+	f := createFixturesForOrganization_AllWhereUserIsOrgAdmin(ms)
+	ofs := f.Organizations
+	ufs := f.Users
+
+	allOrgIDs := []int{ofs[0].ID, ofs[1].ID, ofs[2].ID, ofs[3].ID, ofs[4].ID}
+
+	tests := []struct {
+		name       string
+		user       User
+		wantOrgIDs []int
+		wantErr    string
+	}{
+		{name: "SuperAdmin", user: ufs[0], wantOrgIDs: allOrgIDs},
+		{name: "SalesAdmin", user: ufs[1], wantOrgIDs: allOrgIDs},
+		{name: "NoRole", user: ufs[2], wantOrgIDs: []int{}},
+		{name: "SingleAdmin", user: ufs[3], wantOrgIDs: []int{ofs[3].ID}},
+		{name: "DoubleAdmin", user: ufs[4], wantOrgIDs: []int{ofs[4].ID}},
```

Comment:
Did you intend to make this user admin of two orgs? That's what seems to be implied by "DoubleAdmin".